### DISCUSSION TITLE
Suggestion: pipette tool

### DISCUSSION THREAD

**Discussion by hairintd** [voxeltycoon/issues#731] [UI&UX]
Building can be much simplier and faster with pipette tool. 
Working much like bulldoze tool, except it just switch toolbars to object selected be ready for building. 
Player press button on screen or holds shortcut key (i.e. shift), cursor highlight any buildable object. Player release key - game returns to general mode, like when / pressed twice. When highlighted object clicked - game will switch toolbars and open corresponding windows where clicked object selected and ready to be built. If for some reason clicked object not allowed to be built - warning sound played or message shown with description why object cannot be selected to be built.

**Comment by brechin** [comment]
@hairintd I've published an initial version of a mod that does something along these lines: https://steamcommunity.com/sharedfiles/filedetails/?id=2484965307

Plenty of work to do to make it fully functional, but it's coming along.
